adaptors/salesforce.md

```diff
@@ -182,8 +182,10 @@ A "Salesforce" Credential record should include:
   you cannot find "security token" in your inbox, see below for how to reset
   it.)
 - Login URL: If a production system, use `https://login.salesforce.com/` (unless
-  you have a custom domain `https://domainName.salesforce.com/`). For sandbox
-  environments, `https://test.salesforce.com/`.
+  you have a custom domain `https://domainName.my.salesforce.com/`). For sandbox
```

Comment:
I can't make head nor tail of this :see_no_evil: 

Can I suggest:

For a production system, use `https://login.salesforce.com`. For sandbox environments, use https://test.salesforce.com. For custom domains, use `<my-domain>.my.salesforce.com` 

I don't think there's any point linking to that docs page. You need a PHD to understand it.

Reply:
🤣 🤣 🤣 , Okay updated the instruction